acme/acme/client.py

```diff
@@ -118,7 +130,8 @@ def deactivate_registration(self, regr):
         :rtype: `.RegistrationResource`
 
         """
-        return self.update_registration(regr, update={'status': 'deactivated'})
+        return self.update_registration(regr, messages.Registration.from_json(
+            {"status": "deactivated"}))
```

Comment:
Ah, sorry, I jumped the gun. This change was not effective; `contact` is still serialized.

Reply:
Ok here I really do not get it.

```python
>>> messages.UpdateRegistration(**dict(messages.Registration.from_json({"status":"deactivated"})))
UpdateRegistration(key=None, contact=(), agreement=None, status='deactivated', terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None, resource='reg')
>>> messages.UpdateRegistration(**dict({"status":"deactivated"}))
UpdateRegistration(key=None, contact=(), agreement=None, status='deactivated', terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None, resource='reg')
```